docs/ota-client-guide/modules/ROOT/pages/metadata-expiry.adoc

```diff
@@ -7,20 +7,19 @@ We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}
 ====
 endif::[]
 
-
-Once you xref:rotating-signing-keys.adoc[take the keys for signing metadata offline], you need to be aware of when this metadata expires. You need to refresh the expiry date before it is reached, otherwise you won't be able to push updates. You can also define your own expiry dates when you take your keys offline.
+If you xref:rotating-signing-keys.adoc[take the keys for signing metadata offline], you need to refresh the metadata expiry date before it is reached. Otherwise, you won't be able to push updates. You can also define your own expiry dates when you take your keys offline.
 
 The default expiry dates are as follows:
 
-* For `targets.json`, the expiry date is **31 days** from when the metadata was last updated.
-* For `root.json`, the expiry date is **365 days** from when the metadata was last updated.
+* For `targets.json`, the expiry period is **31 days** from the last time a software update was created.
+* For `root.json`, the expiry period is **365 days** from the last time a software update was created.
```

Comment:
This is more wrong than the previous version. 

* An expiry isn't a period, it's a specific date/time. When you use garage-sign to create or modify metadata, it automatically updates the expiry date to a date that is 31/365 days in the future from the moment the command is run. Before the expiry date, the metadata is valid; after the expiry date it's expired.
* Creating a software update doesn't do anything at all to Uptane metadata.

Reply:
Changed. Hope I got it right.